hello-slack/app.py

```
@@ -0,0 +1,378 @@
+)
+
+# OpenAIクライアントを初期化
+if openai_api_key:
+    client = OpenAI(api_key=openai_api_key)
+else:
+    client = None
+    logging.warning("OPENAI_API_KEY is not set. ChatGPT integration will be disabled.")
+
+
+@app.event("app_mention")
+def handle_app_mention(event, say, context, client): # client を引数に追加 (files_infoのため)
```

Ruff (0.8.2)

`handle_app_mention` is too complex (37 > 10)

(C901)